main/acle.md

```diff
@@ -925,8 +925,9 @@ and:
 to the more specific header files below. These intrinsics are in the
 C implementation namespace and begin with double underscores. It is
 unspecified whether they are available without the header being
-included. The `__ARM_ACLE` macro should be tested before including the
-header:
+included. The `__ARM_ACLE` macro can be tested before including the header.
+However, this is not recommended for cases where target feature selection is
+overridden within code, for example, when using #function-multi-versioning.
```

Comment:
Agreed, but also: `__ARM_ACLE` is a special case in that, like `__ARM_NEON_SVE_BRIDGE`, it describes the presence of a header file rather than a specific architecture feature.  So I think the original text was ok too (and so is your suggestion).

For the feature macros, maybe “is available in a normal unannotated function” would be more specific than “is available”.

Reply:
So this was suggested by @paulwalker-arm 

The `XYZ` macro can be tested before including the header. However, this is not recommended for cases where target feature selection is overridden within code, for example, when using #function-multi-versioning.
From another previous asked change.
 
I have no preference, but would be nice to agree in what to write here, before I change again.  The sentence you see here is a mix of previous interaction that we agree we should replace taking into account FMV.

Reply:
I'm not sure that wording is appropriate for `__ARM_ACLE` though (specifically that macro, not others).  There is no change in features that would affect whether the compiler provides `<arm_acle.h>`.  Intrinsics within `<arm_acle.h>` that depend on features have their own `__ARM_FEATURE_*` macro.